src/app/[locale]/(home)/cart/_components/clear-cart-button.tsx

```
@@ -0,0 +1,59 @@
+"use client";
+
+import { useState } from "react";
+import { toast } from "sonner";
+import { Button } from "@components/ui/button";
+import { BrushCleaning } from "lucide-react";
+import { clearCartAction } from "@lib/apis/cart/clear-logged-cart.api";
+import { useRouter } from "@i18n/navigation";
+import { ApiResponse, CartErrorResponse } from "@lib/types/components/cart";
```

ESLint

[error] 'CartErrorResponse' is defined but never used.

(@typescript-eslint/no-unused-vars)